.github/workflows/ci-gh.yml

```diff
@@ -0,0 +1,57 @@
+name: CI (GPU)
+
+concurrency:
+  group: ci-gpu-on-${{ github.event_name }}-from-${{ github.ref_name }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches-ignore:
+      - gh-pages  # deployment target branch (this workflow should not exist on that branch anyway)
+  pull_request:
+    branches-ignore:
+      - gh-pages  # deployment target branch (this workflow should not exist on that branch anyway)
+
+jobs:
+  build:
+    permissions: read-all
+    runs-on: ubuntu-latest
+    container:
+      options: -u root
+      image: rapidsai/devcontainers:23.06-cpp-cuda11.8-mambaforge-ubuntu22.04
+      env:
+        PYTHONDONTWRITEBYTECODE: 1
+        SCCACHE_REGION: us-east-2
+        SCCACHE_BUCKET: rapids-sccache-devs
+        SCCACHE_S3_KEY_PREFIX: legate-cunumeric-dev
+        VAULT_HOST: https://vault.ops.k8s.rapids.ai
+        GH_TOKEN: "${{ secrets.PERSONAL_ACCESS_TOKEN }}"
```

Comment:
Secrets will not work in the pull request event I think. Does it work for you in pull request events?

Reply:
Yes, I agree. The one time key/hash/token GITHUB_TOKEN is all you get in pull requests and it is not enough to make use of GH CLI tools.

Reply:
Well, we will need to enable secrets in builds by either changing the build type or by copying PRs into branches.

Reply:
Secrets no longer required in pull requests since pull requests will be merged in to a separate branch and tested.